packages/preview/lacy-ubc-math-project/0.1.0/help.typ

```diff
@@ -0,0 +1,123 @@
+#let font-size = 10pt
+
+#let sections = (
+  "introduction",
+  "getting-started",
+  "setup",
+  "author",
+  "math",
+  "drawing",
+  "question",
+  "solution",
+  "caveats",
+)
+
+#let get-orientation(tag) = {
```

Comment:
This is not the intended use for labels, and this will produce a warning if users try to display some manual sections (because the same label is used more than once). A nicer alternative hack to achieve what you are doing here would be to have a custom language for raw blocks that indicates which layout to use, that you replace with `typst` in the end.
Otherwise, simply having a function in which to wrap raw blocks would be the clean approach here.